TinyNvidiaUpdateChecker/MainConsole.cs

```diff
@@ -554,6 +557,33 @@ private static void GpuInfo()
                 Console.WriteLine(ex.ToString());
             }
 
+            // determine whether or not we need a DCH driver
+            try
+            {
```

Comment:
If you look at the existing code, it doesn't have newlines for the `{ }` characters

For example, you wrote like this
```csharp
}
else
{
```
I write like this

```csharp
} else {
```